SECURITY.md

```diff
@@ -0,0 +1,203 @@
+# Security Release Process
+
+Rook is a large growing community of storage providers, contributors, and users. The Rook community
+has adopted this security disclosure and response policy to ensure we responsibly handle critical
+issues.
+
+This policy is adapted from the policies of the following CNCF projects:
+
+* [Envoy](https://github.com/envoyproxy/envoy)
+* [CoreDNS](https://github.com/coredns/coredns)
+
+The Rook community credits and appreciates the example and security best practices that they have
+published openly.
+
+## Product Security Team (PST)
+
+Security vulnerabilities should be handled quickly and sometimes privately. The primary goal of this
+process is to reduce the total time users are vulnerable to publicly known exploits.
+
+The Product Security Team (PST) is responsible for organizing the entire response including internal
+communication and external disclosure.
+
+The initial Product Security Team will consist of the set of maintainers that volunteered.
```

Comment:
How about identifying a security representative per stable or beta storage provider? The security issues will certainly almost always be specific to one storage provider.

Reply:
this sounds like a good idea.  I'll reword this to something like:

> The initial Product Security Team will consist of the set of maintainers that volunteered.  Every beta or stable storage provider MUST have a representative on the PST.